packages/web-storage/.eslintignore

```diff
@@ -0,0 +1,7 @@
+# npm
+node_modules
+package-lock.json
+npm-debug.log*
+
```

Comment:
We will have to update this file to ignore pnpm-lock file as well. See if anything else also needs to be added here.